src/pages/home/report/ReportActionsView.js

```diff
@@ -277,6 +271,22 @@ class ReportActionsView extends React.Component {
         Report.openReport(this.props.report.reportID);
     }
 
+    /**
+     * @returns {String|null}
+    */
+    computeNewMarkerReportActionID() {
+        let newMarkerReportActionID = null;
+        if (ReportUtils.isUnread(this.props.report)) {
+            const newMarkerIndex = _.findLastIndex(this.sortedReportActions, reportAction => (
+                (reportAction.reportActionTimestamp || 0) > this.props.report.lastReadTimestamp
+            ));
+            newMarkerReportActionID = _.has(this.sortedReportActions[newMarkerIndex], 'reportActionID')
+                ? this.sortedReportActions[newMarkerIndex].reportActionID
+                : null;
```

Comment:
What are we guarding against in this case? Why would any action not have a `reportActionID`?

Reply:
If, for example, the sortedReportActions is an empty array for some reason (e.g. we are offline and don't have the report actions).